Threat model analysis and blog post review PR

oduameh opened the pull request:
## Content

<!-- Explain the reason for this change, if a feature is added, a bug is fixed, ... -->

This PR includes...

## Pre-submit checklist

- Branch
  - [ ] Tests are provided (if possible)
  - [ ] Crates versions are updated (if relevant)
  - [ ] CHANGELOG file is updated (if relevant)
  - [ ] Commit sequence broadly makes sense
  - [ ] Key commits have useful messages
- PR
  - [ ] No clippy warnings in the CI
  - [ ] Self-reviewed the diff
  - [ ] Useful pull request description
  - [ ] Reviewer requested
- Documentation
  - [ ] Update README file (if relevant)
  - [ ] Update documentation website (if relevant)
  - [ ] Add dev blog post (if relevant)

## Comments

<!-- Some optional comments about the PR, such as how to run a command, or warnings about usage, .... -->

## Issue(s)

<!-- The issue(s) this PR relates to or closes -->

Relates to #YYY or Closes #YYY

jpraynaud commented on docs/website/blog/2024-12-17-distribution-2450.md:
```diff
@@ -9,23 +9,23 @@ tags: [release, distribution, 2450]
-- the **pre-release-preview** network: there is nothing to do at the moment.
-- the **release-preprod** network: you can upgrade the binary of your signer node to the version `0.2.221`, no configuration update is needed.
-- the **release-mainnet** network: you can upgrade the binary of your signer node to the version `0.2.221`, no configuration update is needed.
+- **Pre-release-preview** network: no action is required at this time
+- **Release-preprod** network: upgrade your signer node binary to version `0.2.221`. No configuration updates are required
+- **Release-mainnet** network: upgrade your signer node binary to version `0.2.221`. No configuration updates are required.
```
```suggestion
- **pre-release-preview** network: no action is required at this time
- **release-preprod** network: upgrade your signer node binary to version `0.2.221`. No configuration updates are required
- **release-mainnet** network: upgrade your signer node binary to version `0.2.221`. No configuration updates are required.
```

jpraynaud commented on docs/website/blog/2024-12-17-era-switch-pythagoras.md:
```diff
@@ -7,45 +7,47 @@ tags: [era, switch, thales, pythagoras]
-- `0.2.200`
-
-All the other versions are not compatible with the new era and must be updated.
+- `v.0.2.221`
+- `v.0.2.209`
+- `v.0.2.200`
```
```suggestion
- `0.2.221`
- `0.2.209`
- `0.2.200`
```

jpraynaud commented on docs/website/blog/2024-12-17-era-switch-pythagoras.md:
```diff
@@ -7,45 +7,47 @@ tags: [era, switch, thales, pythagoras]
-The era switch to `Pythagoras` plan is the following:
+#### Era switch plan for `Pythagoras`
+
+- **Pre-release-preview** network:  
```
```suggestion
- **pre-release-preview** network:  
```

jpraynaud commented on docs/website/blog/2024-12-17-era-switch-pythagoras.md:
```diff
@@ -7,45 +7,47 @@ tags: [era, switch, thales, pythagoras]
-- [ ] Era switch on `release-mainnet`:
-  - [ ] Create the era switch transaction (**to be done, expected early January 2025**).
-  - [ ] Era switch to `Pythagoras`.
+- **Release-preprod** network:  
```
```suggestion
- **release-preprod** network:  
```

jpraynaud commented on docs/website/blog/2024-12-17-era-switch-pythagoras.md:
```diff
@@ -7,45 +7,47 @@ tags: [era, switch, thales, pythagoras]
+  - [x] Create the era switch transaction (done at epoch `184`) 
+  - [x] Complete the era switch to `Pythagoras` at the transition to epoch `186`
+
+- **Release-mainnet** network:  
```
```suggestion
- **release-mainnet** network:  
```

jpraynaud commented on docs/website/blog/2024-12-17-era-switch-pythagoras.md:
```diff
@@ -7,45 +7,47 @@ tags: [era, switch, thales, pythagoras]
+All other versions are **not** compatible with the new era and must be updated.
 :::
 
 :::tip
```
```suggestion
:::tip

```

jpraynaud commented on docs/website/blog/2024-12-17-era-switch-pythagoras.md:
```diff
@@ -7,45 +7,47 @@ tags: [era, switch, thales, pythagoras]
+- `v.0.2.209`
+- `v.0.2.200`
 
+All other versions are **not** compatible with the new era and must be updated.
```
```suggestion
All other versions are **not** compatible with the new era and must be updated.

```

jpraynaud commented on docs/website/blog/2024-12-17-era-switch-pythagoras.md:
```diff
@@ -7,45 +7,47 @@ tags: [era, switch, thales, pythagoras]
+We have introduced the **Pythagoras era** in the Mithril networks. The switch to `Pythagoras` is a significant milestone that brings new features and improvements to the Mithril protocol.
 
 :::danger
+**Mithril signer versions** compatible with the new `Pythagoras` era are:
```
```suggestion

**Mithril signer versions** compatible with the new `Pythagoras` era are:
```

jpraynaud commented on docs/website/blog/2024-12-17-era-switch-pythagoras.md:
```diff
@@ -7,45 +7,47 @@ tags: [era, switch, thales, pythagoras]
-We use the **era switch mechanism to introduce breaking changes** in the Mithril protocol. The new features and improvements are not backward compatible with the previous era. This means that a large majority of at least `95%` of the stake running the new version is required to activate the new era. More information about the era switch mechanism can be found in the [Mithril Network Upgrade Strat ...
+If you have any questions or need assistance, feel free to contact the team on the [Discord channel](https://discord.gg/5kaErDKDRq).
 
 :::
```
```suggestion

```

jpraynaud reviewed (commented):
@oduameh I left few comments below.

Also, could you:
- synchronize your fork so that we can fix some conflicts on the branch
- sign your commits as explained [here](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (otherwise we can't merge the PR)

olgahryniuk reviewed (approved):
@oduameh, please address JP's review suggestions. Particularly, I believe we want network names to be lowercase  (eg, `release-preprod` instead of `Release-preprod`). I made some more edits directly.

oduameh commented:
Hi @jpraynaud I just addressed the issues you raised just working on signing the commit

jpraynaud commented:
> Hi @jpraynaud I just addressed the issues you raised just working on signing the commit

Hi @oduameh, thanks! No need to repush the branch as I handled to sign them this time. But for the next PR you will need to sign the commits.

Also, I have given you write rights on the repository, so you can create a branch on the repository without going through your fork (which will simplify the process of merging) :+1:

oduameh commented:
> > Hi @jpraynaud I just addressed the issues you raised just working on signing the commit
> 
> Hi @oduameh, thanks! No need to repush the branch as I handled to sign them this time. But for the next PR you will need to sign the commits.
> 
> Also, I have given you write rights on the repository, so you can create a branch on the repository without going through your fork (which will simplify the process of merging) 👍

Thanks @jpraynaud
